espresso/devnet-tests/devnet_tools.go

```diff
@@ -295,18 +295,46 @@ func (d *Devnet) SubmitL2Tx(applyTxOpts helpers.TxOptsFn) (*types.Receipt, error
 
 // Waits for a previously submitted transaction to be confirmed by the verifier.
 func (d *Devnet) VerifyL2Tx(receipt *types.Receipt) error {
-	ctx, cancel := context.WithTimeout(d.ctx, 2*time.Minute)
-	defer cancel()
+	// Use longer timeout and retry logic in CI environments due to Espresso processing delays
+	maxRetries := 1
```

Comment:
Is there a reason we have a retry loop here instead of just increasing the timeout?